code42/py42
# Prevent alert note clearing

### Description of Change ###

Ignores `note` parameter for `alerts.update_state()` when given `None`. Before, it would default to the empty string and therefore clear the existing note.

### Issues Resolved ###
no filed issue

### Testing Procedure ###

Before:
Let `alert_id` refer to the ID of an alert in Code42 where the `note.message` is set.
Call `sdk.alerts.update_state("PENDING", [alert_id])`
Notice now that the alert's note.message was cleared.

After:
Let `alert_id` refer to the ID of an alert in Code42 where the `note.message` is set.
Call `sdk.alerts.update_state("PENDING", [alert_id])`
Notice now that the alert's note.message was not affected.

### PR Checklist ###
Did you remember to do the below?
